add deploy to netlify button to each generator

eliwilliamson opened the pull request.

bdougie commented:
This won't work out of the box, we need a template url for each project. The main repo is not a deployable repo in all cases

eliwilliamson commented:
@biilmann per your feedback i did the following:

• made ‘deploy-to-netlify’ button optional/conditional (if set, it is included)
• added a brief blurb describing what ‘deploy to netlify’ does (landing and detail pages)
• added a ‘deploy to netlify’ button to bottom of readme/detail page (with blurb)

LGTY?

@bdougie the template repo is now conditional and customizable (see commit above). i also fixed the height weirdness by making the isotope plugin render with masonry (not standard grid).
